### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by grampassonnia**
- #38 Set up basic routing
- Set up basic landing page
- Connect landing page with upload page
- #39 Create dynamic alert component for upload page
- Connect upload page to analytics page
- Refactor data extraction and define data type for backend transmission (fetch request not implemented yet, waiting for backend permission)
- Set default batch size for file transmission to s3 bucket (still sending to local bucket atm)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://illinkchemipasseràleo
```
Comment (JackBlakeston) [inline .env:1]: cos'è questo?

File: src/App.test.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const uploadButton = screen.getByText('UPLOAD VIDEO');
+  render(<BrowserRouter><App /></BrowserRouter>);
```
Comment (JackBlakeston) [inline src/App.test.tsx:7]: never used browserRouter so I cant really give you much feedback on the router design, I trust you did a good job though

**Review by JackBlakeston — APPROVED** [review]
nice job, lot of code!

File: src/App.test.tsx (hunk 2 above)
Comment (grampassonnia) [reply src/App.test.tsx:7]: This was only to shut the error from the test :D

File: .env (hunk 1 above)
Comment (grampassonnia) [reply .env:1]: Oh sorry, env variable ahahah but since I didn't have it yet I wrote a silly placeholder

**Pull request merged**